server/src/test/java/com/linecorp/centraldogma/server/internal/plugin/PluginTest.java

```diff
@@ -46,9 +46,9 @@
 import com.linecorp.centraldogma.common.Change;
 import com.linecorp.centraldogma.common.Revision;
 import com.linecorp.centraldogma.common.ShuttingDownException;
+import com.linecorp.centraldogma.server.internal.storage.project.DefaultProject;
 import com.linecorp.centraldogma.server.internal.storage.project.DefaultProjectManager;
-import com.linecorp.centraldogma.server.internal.storage.project.Project;
-import com.linecorp.centraldogma.server.internal.storage.project.ProjectManager;
+import com.linecorp.centraldogma.server.storage.project.ProjectManager;
 
 @Ignore("Nashorn raises an AssertionError for an unknown reason.")
 public class PluginTest {
```

Comment:
How about zapping this completely, including the JS-based plugin code?

Reply:
Removed! Thanks! 😃